klips-widget-api/src/main.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+import './style.css';
+import { getParams, parseURLPathnames } from './util/Url';
+import { validateParams } from './util/Config.js';
+
+import ChartApi from './components/Chart/index.js';
+
+import { Params } from './types';
+import { pathNameConfig } from './constants/index.js';
+
+const widgetApi = class WidgetAPI {
```

Comment:
Very minor, but the `WidgetAPI` can be moved to an own file/module.

Reply:
yup, will do thanks